### ISSUE TITLE
IPF color keys might be shifted across crystal Y axis

### ISSUE THREAD

**Issue by hakonanes** [mtex-toolbox/mtex#1245]
Looks like the colors from IPF TSL/HKL color keys of Laue groups (*C2h*, *S6*, *C4h*, *C6h*) are "shifted across the crystal Y axis". I see the same for the HKL key. Not sure if this is an issue for the HSV key, since there isn't a color jump across the crystal Y axis in that key. Unfortunately not sure what the reason is.

**What result do you get**
```matlab
cs = crystalSymmetry('C2h'); % S6, C4h, C6h
ckey = ipfTSLKey(cs);
figure, plot(ckey)
```

![mtex_ipftslkey_c2h](https://user-images.githubusercontent.com/12139781/136068956-57c743fa-5183-41e4-9fab-24e76d62e398.png)
![mtex_ipftslkey_s6](https://user-images.githubusercontent.com/12139781/136070009-bc2c8006-31f7-4f42-bf8c-4560d89c9268.png)
![mtex_ipftslkey_c4h](https://user-images.githubusercontent.com/12139781/136070025-ffda9f99-3d3a-473f-a1a7-ba94369d99a9.png)
![mtex_ipftslkey_c6h](https://user-images.githubusercontent.com/12139781/136070045-deef218f-faa4-457f-80bc-f31093422868.png)

**What MTEX version do you use?**
5.7.0 with MATLAB R2019b.

**Comment by hakonanes** [comment]
Should be fixed in #1261. Thanks!

**Comment by ralfHielscher** [comment]
Hi Håkon,

thank you for reporting and fixing this.

Ralf.
